### PR TITLE
Feature/us15210 culture column

### PR DESCRIPTION
**Pull request by bzmillerboy**
Prototype, do not merge, for the "culture column" section.

### CODE REVIEW

**Comment by bzmillerboy** [comment]
Can I get a review on this PR? This is more less a prototype the content team wants to try out for a couple weeks. We'll have follow up work if it's a success.

File: _assets/javascripts/application.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
 //= require ./vendor/jquery-3.3.1.min
-// require ./vendor/crds-shared-header-v0.7.2.min
-//= require ./lib/set-redirect-url
\ No newline at end of file
+//= require ./vendor/crds-shared-header-v0.7.2.min
+//= require ./lib/set-redirect-url
+//= require ./components/dates.js
```
Comment (tcmacdonald) [inline _assets/javascripts/application.js]: Does this script actually need to be render blocking? If not, it should go in the `application-deferred.js` file

File: _includes/_head.html [hunk 2]
```diff
@@ -26,6 +26,8 @@
   <meta name="theme-color" content="#ffffff">
 
   {% asset application.scss !integrity %}
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.22.2/moment.min.js"></script>
```
Comment (tcmacdonald) [inline _includes/_head.html]: This file should be added to the `_assets/javascripts/vendor` directory and reflected in the manifest file so we can reduce the number of HTTP requests on the page.

**Comment by bzmillerboy** [comment]
@tcmacdonald Review suggestions made

**Pull request merged**